telegram/base.py

```diff
@@ -84,7 +76,7 @@ def to_dict(self):
     def __eq__(self, other):
         if isinstance(other, self.__class__):
             return self._id_attrs == other._id_attrs
-        return super(TelegramObject, self).__eq__(other)
+        return super(TelegramObject, self).__eq__(other)  # pylint: disable=E1101
```

Comment:
Out of scope of this PR? You added the comment though... I'm not asking you to go back and find all our pylint lines, I'm just thinking that we should start slowly switching over so we make it easier for ourselves in the future.